core/block_render_svg_vertical.js

```diff
@@ -730,6 +747,31 @@ Blockly.BlockSvg.prototype.renderFields_ =
   return this.RTL ? -cursorX : cursorX;
 }; /* eslint-enable indent */
 
+/**
+ * Add a vertical separator line SVG after the current field.
+ * @param {Blockly.Field} field - the field after which to place the line.
```

Comment:
Notes on annotations:

- If field is non-nullable, this is `@param {!Blockly.Field}`.
- No hyphen between the argument name and the description.  (JSDoc [accepts](http://usejsdoc.org/tags-param.html) hyphens and strips them out correctly, as far as I can tell, but that's not the [standard](https://developers.google.com/blockly/guides/modify/web/style-guide#functions) that blockly/scratch blocks code uses.)
- Capitalize the first letter of the description.